build/CoreText/dll/CoreText.def

```diff
@@ -204,6 +204,9 @@ LIBRARY CoreText
         CTFrameGetLineOrigins
         CTFrameDraw
         CTFrameGetTypeID
+        CTFrameDrawUninverted
```

Comment:
All private methods should be _, just to be aware of code that takes dependency on them.